packages/circuits/test/proveData.test.ts

```diff
@@ -81,6 +82,18 @@ const genProofAndVerify = async (
     return { isValid, proof, publicSignals }
 }
 
+const genReplField = (value: number | bigint, length: number) => {
```

Comment:
actually the repField could be `FIELD_COUNT-SUM_FIELD_COUNT` different data.
In the previous test (though might not be a good solution)
`Array(SUM_FIELD_COUNT).fill(6)`
 we know that it is an array fills with `6`

but in this function it might be confusing that we only insert one `value` into the array
I would suggest if we make it the function `genReplField`
we should let it be 
```ts
const genReplField = (values: (number | bigint)[]) => {}
```
and the length is `FIELD_COUNT - SUM_FIELD_COUNT` (could be fixed in the file)
then it can take the array input like `Array(SUM_FIELD_COUNT).fill(6)`